### DISCUSSION TITLE
AWS CLI comment stop-db-cluster can custom the restart day ?

### DISCUSSION THREAD

**Discussion by EmberLeung** [aws/aws-cli#6911] [Q&A] [answered]
Hello everyone,

When I stop the cluster, it will restart at the default day is 7 days.
I would like to custom the restart day when I stop the cluster.
Do AWS CLI command "stop-db-cluster" can custom the restart day ?

Thank you very much

**Comment by tim-finnigan** [comment, marked as answer]
Hi @EmberLeung thanks for reaching out. I don't believe there is a way to customize the restart day in this case. But you could consider using a service such as AWS Lambda to automatically restart clusters on a schedule. This article can help guide you through a similar process: https://aws.amazon.com/blogs/database/schedule-amazon-rds-stop-and-start-using-aws-lambda/

**Reply by EmberLeung** [reply]
OK, Thank you very much !
I try to use that :D
